### PR TITLE
Py2to3

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: kitsune/users/templatetags/jinja_helpers.py [hunk 1]
```diff
@@ -84,16 +84,15 @@ def public_email(email):
...
 
 
 @library.global_function
 def user_list(users):
     """Turn a list of users into a list of links to their profiles."""
-    link = u'<a class="user secondary-color" href="%s">%s</a>'
-    list = u", ".join(
-        [link % (escape(profile_url(u)), escape(display_name(u))) for u in users]
-    )
+    link = '<a class="user" href="%s">%s</a>'
```
Comment (LeoMcA) [inline kitsune/users/templatetags/jinja_helpers.py:93]: Did we mean to lose `secondary-color` here?

File: kitsune/wiki/tests/test_templates.py [hunk 2]
```diff
@@ -874,11 +875,25 @@ def test_new_revision_POST_document_with_current(self, get_current):
...
+            body=READY_FOR_REVIEW_EMAIL_CONTENT % {
+                'user': self.user.profile.name,
+                'title': self.d.title,
+                'slug': self.d.slug,
+                'new_id': new_rev.id,
+                'summary': new_rev.summary,
+                'diff': diff,
+                'watcher': reviewable_watch.pk,
+                'secret': reviewable_watch.secret
+>>>>>>> c8d298c22... 2to3 unicode fix.
```
Comment (LeoMcA) [inline kitsune/wiki/tests/test_templates.py:890]: merge conflict

File: kitsune/wiki/tests/test_templates.py [hunk 3]
```diff
@@ -889,6 +904,18 @@ def test_new_revision_POST_document_with_current(self, get_current):
...
+                self.d.title, new_rev.creator),
+            body=DOCUMENT_EDITED_EMAIL_CONTENT % {
+                'user': self.user.profile.name,
+                'title': self.d.title,
+                'slug': self.d.slug,
+                'watcher': edit_watch.pk,
+                'secret': edit_watch.secret,
+                'summary': new_rev.summary,
+                'diff': diff,
+>>>>>>> c8d298c22... 2to3 unicode fix.
```
Comment (LeoMcA) [inline kitsune/wiki/tests/test_templates.py:918]: merge conflict

File: kitsune/wiki/tests/test_templates.py [hunk 4]
```diff
@@ -1553,6 +1581,13 @@ def test_approve_revision(self, get_current, reviewed_delay):
             body=expected_body,
             to=["joe@example.com"],
         )
+=======
+        attrs_eq(mail.outbox[0],
+                 subject=('{0} ({1}) has a new approved revision ({2})'
+                          .format(self.document.title, self.document.locale, self.user.username)),
+                 body=expected_body,
+                 to=['joe@example.com'])
+>>>>>>> c8d298c22... 2to3 unicode fix.
```
Comment (LeoMcA) [inline kitsune/wiki/tests/test_templates.py:1590]: merge conflict

File: kitsune/users/templatetags/jinja_helpers.py (hunk 1 above)
Comment (akatsoulas) [reply kitsune/users/templatetags/jinja_helpers.py:93]: nope

**Pull request merged**